proter/src/main/scala/com/workflowfm/proter/flows/Flow.scala

```diff
@@ -330,3 +333,57 @@ class FlowSimulationGenerator(baseName: String, flow: Flow) extends SimulationGe
     new FlowSimulation(name, manager, flow.copy())
   }
 }
+
+  /**
+    * Factory for making random flows given some parameters
+    *
+    * @param andProbability Probability that a flow node is an AND. Must in range [0,1]
```

Comment:
- "Must **be** in range..."
- Change `AND` --> `[[And]]` - this will add a link to the actual constructor
- You need to point out that the inverse probability yields a `Then`.